### DISCUSSION TITLE
Load balance

### DISCUSSION THREAD

**Discussion by viniciusverasdossantos** [microsoft/aspire#4363] [Q&A]
I would like to test the load balance of my api by uploading several instances of it to simulate distributed caching, etc. Is there anything that helps with Aspire? Any tips?

**Comment by lorenzoponsbarber** [comment]
Hello Vinicius,

Of course! this is the option:

```
builder.AddProject<Projects.My_Project_API>("my-project-api")
    .WithReplicas(3);
```

Greetings
